doc/examples/pynest_microcircuit_link.rst

```diff
@@ -0,0 +1,2 @@
+.. include:: ../../pynest/examples/Potjans_2014/README.rst
```

Comment:
I think this should be copied too, so that we can stay in the doc tree with this link and just point to `Potjans_2014/README.rst`. If the `pynest/examples` is moved, we only want to fix the one source path in `conf.py`

Reply:
If this is also copied, a documentation page is also built from it. Then this page exists twice. With each building process it will be copied and converted by sphinx_gallery. This will be sorted in the documentation.

Reply:
If the duplication of the readme leads to the page existing twice, it is ok to use the original version.
Thanks for the background!
:+1: